README.md

```diff
@@ -111,7 +111,9 @@ Check the [Install Guide](GUIDE.md) for installation instructions.
 | WORKER_MEMORY               | memory for worker VM, default: 8892MB                                                                                                       |
 | PUBLIC_CONTAINER_REGISTRIES | comma-separated list of registries that do not require authentication for pulling assisted installer images                                 |
 | CHECK_CLUSTER_VERSION       | If "True", the controller will wait for CVO to finish                                                                                       |
-| ENABLE_KUBE_API             | If set, deploy assisted-service with Kube API controllers (minikube only)                                                                                     |
+| ENABLE_KUBE_API             | If set, deploy assisted-service with Kube API controllers (minikube only)                                                                   |
+| USER_MANAGED_NETWORKING     | If true set cluster user_managed_networking to True, default: False                                                                         |
+| HIGH_AVAILABILITY_MODE      | Set cluster high_availability_mode. Available values: Full or None, default: Full                                                           |
```

Comment:
Isn't it an implementation detail to how we do our tests?
I think that because it's an implementation detail for the UI (and user doesn't set it by himself) then maybe we shouldn't support changing it in our testing infrastructure

Reply:
Maybe better to call it SINGLE_NODE? Or why not to use same logic as now NUM_MASTER=1, why we should have another field for it